process-migration-service/frontend/cypress/integration/spec.js

```diff
@@ -0,0 +1,5 @@
+describe('First Test', function() {
+  it('Example Test', function() {
+    expect(true).to.equal(true)
```

Comment:
Please put here a meaningful check of the started web app, at least simple cy.visit().

Reply:
No, make sure you start PIM UI before this test is executed. Here you just invoke the default endpoint of PIM UI to verify all is configured correctly.

Reply:
then i need config.yml with proper setup..is that okay to commit?

Reply:
You should not need it.
The order of steps we're doing is:
1. navigate to ./process-migration-service/frontend
2. run `npm install`
3. run `npm run build`
4. run `npm run start`

And PIM UI is then started at localhost:3000, I've just verified these steps.

`config.yml` file is required by Thorntail, but here we should be able to test without thorntail integration.

Reply:
ok

Reply:
You might be fighting existing failsafe configuration in the pom.xml - symptom would be `npm test:it` works fine when invoked in terminal, but failing during maven build. Just fyi that it might happen.

Reply:
> You should not need it.
> The order of steps we're doing is:
> 
> 1. navigate to ./process-migration-service/frontend
> 2. run `npm install`
> 3. run `npm run build`
> 4. run `npm run start`
> 
> And PIM UI is then started at localhost:3000, I've just verified these steps.
> 
> `config.yml` file is required by Thorntail, but here we should be able to test without thorntail integration.
i see no 'npm run start' if i add in pom.xml (as execution) , this will be called everytime..pls suggest me on this .

Reply:
start-server-with-test i hope this will suffice :_)